src/network/serveropcodes.cpp

```diff
@@ -82,7 +82,7 @@ const ToServerCommandHandler toServerCommandTable[TOSERVER_NUM_MSG_TYPES] =
 	{ "TOSERVER_DAMAGE",                   TOSERVER_STATE_INGAME, &Server::handleCommand_Damage }, // 0x35
 	null_command_handler, // 0x36
 	{ "TOSERVER_PLAYERITEM",               TOSERVER_STATE_INGAME, &Server::handleCommand_PlayerItem }, // 0x37
-	{ "TOSERVER_RESPAWN",                  TOSERVER_STATE_INGAME, &Server::handleCommand_Respawn }, // 0x38
+	null_command_handler,
```

Comment:
we agreee that the protocol bump is a breakage from old client and 5.10 only supports 5.10 ?

Reply:
As explained above, this PR doesn't break backwards compatibility. Feel free to test.

Reply:
ok i understood, you send the flexible death screen everytime and client is not using (on 5.9) the embeded one. It's fine.